src/app/shared-module/validation/number.ts

```diff
@@ -0,0 +1,20 @@
+import { AbstractControl, ValidationErrors, ValidatorFn } from '@angular/forms';
+
+export class NumberValidator {
+  static number(min?: number, max?: number): ValidatorFn {
+    return (control: AbstractControl): ValidationErrors | null => {
+      const value = Number(control.value);
+
+      if (isNaN(value)) {
+        return { notNumber: true };
+      }
+      if (min && value < min) {
+        return { numberLessThanMin: min };
+      }
+      if (max && value > max) {
+        return { numberMoreThanMax: max };
+      }
+      return null;
+    };
+  }
```

Comment:
Validators contain methods for the similar logic you've implemented here. Check for the `settingsValidators` in #12. It's not that critical but the less code, the better, right?

Reply:
I agree with you)